spec/models/seed_spec.rb

```diff
@@ -152,4 +152,32 @@
     end
   end
 
+  context 'photos' do
+    before(:each) do
+      @seed = FactoryGirl.create(:seed)
+      @photo = FactoryGirl.create(:photo)
```

Comment:
It certainly is (and can make things a lot neater with complex graphs of objects), but it might be even better to consider a "seed-with-complete-data" factory that has some smart defaults.

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations